sdk/eventhub/Microsoft.Azure.EventHubs.Processor/src/AzureBlobLease.cs

```diff
@@ -11,48 +12,66 @@ class AzureBlobLease : Lease
 	{
         readonly bool isOwned;
 
-	    // ctor needed for deserialization
-	    internal AzureBlobLease()
+        readonly TimeSpan LeaseDuration;
+
+        private bool IsInfiniteLease => this.LeaseDuration > TimeSpan.FromSeconds(60);
```

Comment:
>IsInfiniteLease [](start = 21, length = 15)

'Infinite' term doesn't suite well with this feature since it is not infinite actually. You are adding some sort of buffer to each lease.